atam4j/src/test/java/me/atam/atam4j/configuration/ConfigLoaderTest.java

```diff
@@ -7,9 +8,45 @@
 public class ConfigLoaderTest {
 
     @Test
-    public void givenAValidYAMLConfigFile_whenGetConfigInvoked_thenValidPOJOReturned(){
-        ConfigLoader<DummyConfiguration> dummyConfigurationConfigLoader = new ConfigLoader<>(DummyConfiguration.class, "config-loader-test.yml");
+    public void givenAValidYAMLConfigFile_whenGetConfigInvoked_thenValidPOJOReturned() {
+        ConfigLoader<DummyConfiguration> dummyConfigurationConfigLoader = new ConfigLoader<>(
+                DummyConfiguration.class,
+                "config-loader-test.yml"
+        );
         assertThat(dummyConfigurationConfigLoader.getTestConfig().getSomeDummyAttribute(), CoreMatchers.is("someValue"));
     }
 
+    @Test
```

Comment:
I would prefer to have the tests use a single config file - `config-loader-test.yml` instead of a 2nd `config-loader.test-env.yml`. 
Reasons:
* Simpler with a single file to manage
* Both files currently using the same attribute `someDummyAttribute` which can be a source of confusion. Certainly confused me in the first look and then I looked again.